projects/valtimo/components/src/lib/components/view-content/view-content.service.ts

```diff
@@ -32,10 +32,19 @@ export class ViewContentService {
   }
 
   public get(value: any, definition: any) {
+    const separator = ':'
     if (!definition.viewType) {
       definition.viewType = typeof value;
     }
 
+    if (definition.viewType.includes(separator)) {
```

Comment:
I think this logic is fine, but it would have been easier with .split. Ie:

`
if (definition.viewType.includes(separator)) {
  const separatedViewType = definition.viewType.split(separator);
  definition.format = separatedViewType[1];
  definition.viewType = separatedViewType[0];
}
`

If you keep it like this please change `getIndexSeparator` to `getSeparatorIndex`

Reply:
I have renamed the function name. You might want to display just the time as well and with split it will not just split the first separator, but all of them (hh:mm A)